scripts/replace.js

```diff
@@ -3,11 +3,10 @@ import path from "path";
 
 const args = process.argv.slice(2);
 
-let NETWORK_ENV = "mainnet";
+let NETWORK_ENV = "testnet";
```

Comment:
Is this right? I think this should maybe default to mainnet or be overriden from arguments